hist/hist/src/TH2.cxx

```diff
@@ -84,7 +97,18 @@ TH2::TH2(const char *name,const char *title,Int_t nbinsx,Double_t xlow,Double_t
 
 
 ////////////////////////////////////////////////////////////////////////////////
-/// See comments in the TH1 base class constructors.
+/// Normal constructor for Double_t variable bin size (along X axis) 2-D histograms.
```

Comment:
Constructor for variable bin size (along X axis) 2-D histograms using an input array of type double.